packages/react-composites/tests/browser/chat/fake-adapter/ReadReceipt.test.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatParticipant } from '@azure/communication-chat';
+import { CommunicationIdentifier } from '@azure/communication-common';
+import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { ElementHandle, expect, Page } from '@playwright/test';
+import { sendMessage, waitForMessageDelivered, waitForMessageSeen } from '../../common/chatTestHelpers';
+import { dataUiId, stableScreenshot, stubReadReceiptsToolTip } from '../../common/utils';
+import { buildUrlForChatAppUsingFakeAdapter, DEFAULT_FAKE_CHAT_ADAPTER_ARGS, test } from './fixture';
+
+test.describe('Chat Composite E2E Tests', () => {
+  test('participant can receive message and send readReceipt to message sender', async ({ serverUrl, page }) => {
+    const messageReader = DEFAULT_FAKE_CHAT_ADAPTER_ARGS.remoteParticipants[0];
+    page.goto(
+      buildUrlForChatAppUsingFakeAdapter(serverUrl, {
+        ...DEFAULT_FAKE_CHAT_ADAPTER_ARGS,
+        participantsWithHiddenComposites: [messageReader]
+      })
+    );
+    await setParticipantAbleToSeeMessages(page, messageReader, false);
+
+    const testMessageText = 'How the turn tables';
+    await sendMessage(page, testMessageText);
+    await waitForMessageDelivered(page);
+    expect(await stableScreenshot(page, { stubMessageTimestamps: true })).toMatchSnapshot('sent-messages.png');
+
+    await setParticipantAbleToSeeMessages(page, messageReader, true);
+    await waitForMessageSeen(page);
+
+    await page.locator(dataUiId('chat-composite-message-status-icon')).click();
+    await page.waitForSelector(dataUiId('chat-composite-message-tooltip'));
```

Comment:
use the `waitForSelector` utils wrapper method we have in the utils instead of the page method